Apps/W1/Shopify/app/src/Invoicing/Reports/ShpfySyncInvoicesToShpfy.Report.al

```diff
@@ -94,5 +99,6 @@ report 30117 "Shpfy Sync Invoices to Shpfy"
     internal procedure SetShop(NewShopCode: Code[20])
     begin
         ShopCode := NewShopCode;
+        ShpfyShop.Get(ShopCode);
```

Comment:
Should you do this Get also OnValidate of the requestPage field? Otherwise the check won't be executed for the correct shop?

Reply:
Maybe immediatly after checking if the SHopCode is filled in, you should retrieve shop instead. 

Otherwise this code will break if someone schedules a report as JobQueue.

Since it will be as part of report parameters, but it will never retrieve a Shop Record.